LOG_AUTH facility is deprecated in favor of LOG_AUTHPRIV.

http://sourceforge.net/tracker/?func=detail&aid=3036068&group_id=80387&atid=559616
--- a/mingetty.c	2008-01-18 06:13:07.000000000 -0500
+++ b/mingetty.c	2010-07-28 11:30:51.265400691 -0400
@@ -83,7 +83,7 @@
 	va_list va_alist;
 
 	va_start (va_alist, fmt);
-	openlog (progname, LOG_PID, LOG_AUTH);
+	openlog (progname, LOG_PID, LOG_AUTHPRIV);
 	vsyslog (LOG_ERR, fmt, va_alist);
 	/* no need, we exit anyway: closelog (); */
 	va_end (va_alist);
